tests/cross_functional/stress/cephfs/test_breakpoint.py

```diff
@@ -0,0 +1,38 @@
+import logging
+
+from ocs_ci.framework.pytest_customization.marks import magenta_squad
+from ocs_ci.framework.testlib import E2ETest
+from ocs_ci.helpers.cephfs_stress_helpers import create_cephfs_stress_pod
+
+logger = logging.getLogger(__name__)
+
+
+@magenta_squad
```

Comment:
other than magenta squad do we need any specific markers like stress_test or any tiers ?